static/src/stylesheets/module/_progress.scss

```diff
@@ -0,0 +1,13 @@
+.progress {
+    position: fixed;
+    z-index: 1;
+    height: $gs-baseline / 4;
```

Comment:
So, does this element intercept click / touch events even when it's empty? Maybe it should start out with pointer-events:none or something.